clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -12514,6 +12514,20 @@ def err_sycl_special_type_num_init_method : Error<
 def warn_launch_bounds_is_cuda_specific : Warning<
   "%0 attribute ignored, only applicable when targeting Nvidia devices">,
   InGroup<IgnoredAttributes>;
+def err_registered_kernels_num_of_args : Error<
+  "'__registered_kernels__' atribute must have at least one argument">;
```

Comment:
Misspelled `attribute` here and in the diagnostics below.

Reply:
Thanks, resolved.